### DISCUSSION TITLE
Official community support/help

### DISCUSSION THREAD

**Discussion by Vertux** [TriliumNext/Trilium#3770] [Q&A] [answered]
Maybe I have overlooked the corresponding information, but is there any official place where users can get support ? I am thinking of Matrix, Discord, IRC, reddit (I have seen there is a Trilium subreddit), forum ?

**Comment by sigaloid** [comment, marked as answer]
[There's a Gitter chatroom!](https://gitter.im/trilium-notes/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)

**Comment by zadam** [comment]
The most "official" place is right here - the github discussion.

**Reply by Vertux** [reply]
Thank you, good to know. It's nice to have everything at one place. I guess I have never used the GitHub discussions section yet.
